CalendarApp/BookingRoom/Models/Event/PostingExtension.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Google.Apis.Calendar.v3.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace BookingRoom.Models.Posting
+{
+    public static class CopyExtension
+    {
+        /// <summary>
+        /// Extension method for Google Event class
+        /// </summary>
+        /// <param name="toAddEvent"></param>
```

Comment:
There is no need to add extra definitions if the do not contain any useful info. Pls remove